### ISSUE TITLE
TsInfo rendered extra paths in error when interface or type is exported

### ISSUE THREAD

**Issue by jacketwpbb** [vitejs/vite-plugin-react-pages#125]
```
export interface I {
  a;  1;
}
interface II {
  a;  1;
}
export interface ButtonProp {
  i: I;
  ii: II;
}
```

| name | type                                                                              |
| ---- | --------------------------------------------------------------------------------- |
| i    | import("/home/projects/vitejs-vite-plugin-react-pages-xgedig/src/Button/types").I |
| ii   | II                                                                                |

[stackblitz button demo](https://stackblitz.com/edit/vitejs-vite-plugin-react-pages-xgedig?file=src%2FButton%2Ftypes.ts,package.json,src%2FButton%2FREADME.md&terminal=dev)

**Comment by csr632** [comment]
Got it. Fixing it.

**Comment by csr632** [comment]
Fixed in vite-plugin-react-pages@4.1.1. Notice that stackblitz's npm registery may need some time to sync up the new version. You may need to manually specific `^4.1.1` in your `package.json`.

Here are more examples such as type re-export: https://github.com/vitejs/vite-plugin-react-pages/blob/f50408d04d1bfcd19b0418c06771bd8ffd0c880b/packages/playground/use-theme-doc/pages/components/button/types.ts#L2
